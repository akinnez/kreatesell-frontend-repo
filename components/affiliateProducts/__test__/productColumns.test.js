import productColumns from '../productsColumns';

it('renders a snapshot of productColumns data', () => {
	expect(productColumns()).toMatchSnapshot();
});
