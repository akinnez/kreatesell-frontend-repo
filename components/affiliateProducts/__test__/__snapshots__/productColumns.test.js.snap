// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of productColumns data 1`] = `
Array [
  Object {
    "dataIndex": "product_name",
    "title": "Product",
  },
  Object {
    "dataIndex": "kreator_name",
    "render": [Function],
    "title": "Kreator",
  },
  Object {
    "dataIndex": "product_type_id",
    "render": [Function],
    "title": "Product Type",
  },
  Object {
    "dataIndex": "date_created",
    "render": [Function],
    "sortDirections": Array [
      "descend",
      "ascend",
      "descend",
    ],
    "sorter": [Function],
    "title": "Launch Date",
  },
  Object {
    "dataIndex": "number_of_sales",
    "render": [Function],
    "sortDirections": Array [
      "descend",
      "ascend",
      "descend",
    ],
    "sorter": [Function],
    "title": "No of Sales",
  },
  Object {
    "dataIndex": "",
    "render": [Function],
    "title": "Performance",
  },
  Object {
    "dataIndex": "affiliate_percentage_on_sales",
    "render": [Function],
    "title": "Commission (%)",
  },
  Object {
    "dataIndex": "",
    "render": [Function],
    "title": "Action",
  },
]
`;
